test/e2e/service_bootstrap.py

```diff
@@ -0,0 +1,44 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Bootstraps the resources required to run networkfirewall integration tests.
+"""
+
+import logging
+
+from acktest.bootstrapping import Resources, BootstrapFailureException
+from acktest.bootstrapping.vpc import VPC
+from acktest.bootstrapping.s3 import Bucket
+from e2e import bootstrap_directory
+from e2e.bootstrap_resources import BootstrapResources
+
+def service_bootstrap() -> Resources:
+    logging.getLogger().setLevel(logging.INFO)
+
+    resources = BootstrapResources(
+        SharedTestVPC=VPC(name_prefix="e2e-test-vpc", num_public_subnet=1, num_private_subnet=0),
```

Comment:
can you include `networkfirewall` in the name prefix? it makes it easier to track if things stay dangling in the account

Reply:
done